src/flamenco/repair/fd_repair.c

```diff
@@ -599,24 +591,21 @@ fd_repair_recv_ping(fd_repair_t * glob, fd_gossip_ping_t const * ping, fd_gossip
   fd_hash_copy( &pong->from, glob->public_key );
 
   /* Generate response hash token */
-  fd_sha256_t sha[1];
-  fd_sha256_init( sha );
-  fd_sha256_append( sha, "SOLANA_PING_PONG", 16UL );
-  fd_sha256_append( sha, ping->token.uc,     32UL );
-  fd_sha256_fini( sha, pong->token.uc );
+  // fd_sha256_t sha[1];
+  // fd_sha256_init( sha );
+  // fd_sha256_append( sha, "SOLANA_PING_PONG", 16UL );
+  // fd_sha256_append( sha, ping->token.uc,     32UL );
+  // fd_sha256_fini( sha, pong->token.uc );
+
+  uchar pre_image[FD_PING_PRE_IMAGE_SZ];
+  fd_memcpy( pre_image, "SOLANA_PING_PONG", 16UL );
+  fd_memcpy( pre_image+16UL, ping->token.uc, 32UL);
```

Comment:
```suggestion
  memcpy( pre_image, "SOLANA_PING_PONG", 16UL );
  memcpy( pre_image+16UL, ping->token.uc, 32UL );
```
memcpy is preferred with constant size arguments, especially powers of 2, because it results in better code gen than fd_memcpy (not that it matters in the infrequent operation here)